internal/description/description.go

```diff
@@ -43,6 +44,8 @@ const (
 	DescribeCluster        = "Describe a cluster."
 	ListClusters           = "List clusters for a project."
 	UpdateCluster          = "Update a MongoDB cluster in Atlas."
+	Processes              = "Manage processes for your project."
```

Comment:
Should the language here always refer to `a project`?

Reply:
This format was done as a favour from the docs team, I'll ask them to give this list of descriptions another go